tools/analysisPyXml.py

```diff
@@ -49,7 +49,7 @@ def analysisPyXml(rootPath, ut):
                          '@', '\'\'\'', 'logger', '_logger', 'logging', 'r"""',
                          'pass', 'try', 'except',
                          'if __name__ == "__main__"')) == False:
-                        pattern = "(.*) = ('*')|(.*) = (\"*\")|(.*) = (\d)|(.*) = (-\d)|(.*) = (None)|(.*) = (True)|(.*) = (False)|(.*) = (URL_PREFIX*)|(.*) = (\[)|(.*) = (\{)|(.*) = (\()"  #a='b'/a="b"/a=0
+                        pattern = r"(.*) = ('*')|(.*) = (\"*\")|(.*) = (\d)|(.*) = (-\d)|(.*) = (None)|(.*) = (True)|(.*) = (False)|(.*) = (URL_PREFIX*)|(.*) = (\[)|(.*) = (\{)|(.*) = (\()"  #a='b'/a="b"/a=0
```

Comment:
这里因为 `'\"*\"'` 的存在，修改后的是不等价的，建议修改成这样

```suggestion
                        pattern = r"""(.*) = ('*')|(.*) = ("*")|(.*) = (\d)|(.*) = (-\d)|(.*) = (None)|(.*) = (True)|(.*) = (False)|(.*) = (URL_PREFIX*)|(.*) = (\[)|(.*) = (\{)|(.*) = (\()"""  #a='b'/a="b"/a=0
```

我验证了下，两者是等价的